contracts/gateway/EIP20Gateway.sol

```diff
@@ -964,7 +980,56 @@ contract EIP20Gateway is Gateway {
         message.gasConsumed = initialGas.sub(gasleft());
     }
 
-    /* Private functions */
+    /**
+    * @notice Activate Gateway contract. Can be set only by the
+    *         Organisation address only once by passing co-gateway address.
+    *
+    * @param _coGatewayAddress Address of cogateway.
+    *
+    * @return `true` if value is set
+    */
+    function activateGateway(address _coGatewayAddress)
+        external
+        onlyOrganisation
+        returns (bool)
```

Comment:
Not sure, but I think if you have a multi-line function head, the arguments must also be multi-line.